id: LTR-066
frame: a_motioncam_sensor
color_camera_frame: a_motioncam_color_camera_frame
hardware_trigger: false
calibration_volume_only: true
interreflections_filtering: false
send_point_cloud: false
send_normal_map: true
send_depth_map: true
send_confidence_map: false
send_event_map: false
send_texture: true
send_color_camera_image: true
dense_cloud: false
operation_mode: 1
laser_power: 4095
led_power: 4095
hardware_trigger_signal: 1
sampling_topology: 1
output_topology: 1
camera_coding_strategy: 2
camera_texture_source: 5
shutter_multiplier: 1
scan_multiplier: 1
scanner_coding_strategy: 2
coding_quality: 2
texture_source: 5
trigger_mode: 0
timeout: -1
surface_smoothness: 2
normals_estimation_radius: 0
iso: 100
color_resolution: 1
white_balance: "Internal LED (5000 - 5500K)"
maximum_fps: 20
camera_exposure: 10.24
scanner_exposure: 10.24
confidence: 3
interreflection_filter_strength: 0
exposure: 10.24
gamma: 1
intensity_scale: 0.5